packages/ai-bot/helpers.ts

```diff
@@ -19,7 +19,8 @@ If the user request is unclear, you may ask clarifying questions. \
 You may make multiple function calls, all calls are gated by the user so multiple options can be explored.\
 If a user asks you about things in the world, use your existing knowledge to help them. Only if necessary, add a *small* caveat at the end of your message to explain that you do not have live external data. \
 \
-If you need access to the cards the user can see, you can ask them to attach the cards.';
+If you need access to the cards the user can see, you can ask them to attach the cards. \
+If you encounter JSON structures, please enclose them within backticks to ensure they are displayed stylishly in Markdown.';
```

Comment:
This is enough? 😯

Reply:
For most things, yeah. The models seem pretty tuned to output markdown anyway, so slight pushes will help. No prompting will be 100%, but unless we see decent rates of issues it may be better to see a few issues than the number we may introduce by trying to identify json reliably.

Hard to say how often it'd happen without this, the previous code was explicitly removing any backticks.

Reply:
I've tested it and it looks well